<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAnimal</class>
 <widget class="QDialog" name="AddAnimal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>461</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>200</red>
        <green>16</green>
        <blue>46</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>238</red>
        <green>238</green>
        <blue>238</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>200</red>
        <green>16</green>
        <blue>46</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>238</red>
        <green>238</green>
        <blue>238</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>190</red>
        <green>190</green>
        <blue>190</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>238</red>
        <green>238</green>
        <blue>238</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Egyptienne F</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>420</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="nameLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="typeLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="typeComboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Please Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cat</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dog</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hamster</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fish</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Snake</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelBreed">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Breed</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="breedComboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Please Select a Type</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="sexLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Sex</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="sexComboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Please Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ageLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ageLineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelColour">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Colour</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="colourComboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Please Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Black</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>White</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Brown</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grey</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="sizeLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Size</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="sizeComboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Please Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Small</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Medium</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Large</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>216</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="aggressionLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Aggression Level (Out of 10)</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="10,2">
      <item>
       <widget class="QSlider" name="aggressionSlider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="aggressionLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="attachmentLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Attachment Level (Out of 10)</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="10,2">
      <item>
       <widget class="QSlider" name="attachmentSlider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="attachmentLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelBreed_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Obedience Level (Out of 10)</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="10,2">
      <item>
       <widget class="QSlider" name="obedienceSlider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="obedienceLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelColour_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Cleanliness Level (Out of 10)</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="10,2">
      <item>
       <widget class="QSlider" name="cleanlinessSlider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cleanlinessLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="aggressionLabel_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Comfort with Children (Out of 10)</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="10,2">
      <item>
       <widget class="QSlider" name="childrenComfortSlider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="childrenComfortLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="attachmentLabel_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Loudness Level (Out of 10)</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="10,2">
      <item>
       <widget class="QSlider" name="loudnessSlider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="loudnessLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="ageLabel_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Intelligence Level (Out of 10)</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="10,2">
      <item>
       <widget class="QSlider" name="intelligenceSlider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="intelligenceLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>220</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="energyLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Energy Level</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="energyComboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Please Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Low</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Medium</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>High</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelBreed_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Flat Cost</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="costLineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="sexLabel_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Cost per Year</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="costPerYearLineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelColour_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Hypoallergenic</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="isCrateTrainedRadioButtonYES">
        <property name="text">
         <string>Yes</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="isCrateTrainedRadioButtonNO">
        <property name="text">
         <string>No</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="ageLabel_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Crate Trained</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="isHypoallergenicRadioButtonYES">
        <property name="text">
         <string>Yes</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="isHypoallergenicRadioButtonNO">
        <property name="text">
         <string>No</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="sizeLabel_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Neutered/Spayed</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="isNeuteredRadioButtonYES">
        <property name="text">
         <string>Yes</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_3</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="isNeuteredRadioButtonNO">
        <property name="text">
         <string>No</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_3</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddAnimal</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>648</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddAnimal</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>716</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
